output.proto

```diff
@@ -70,8 +70,13 @@ message SystemHealth {
       string topic = 2;
       string sensor_name = 3;
     }
+    message EdgeNodeInfo {
+      Status status = 1;
+      map<string, SensorInfo> sensors = 4;
+    }
```

Comment:
Why not 
```suggestion
    message EdgeNodeInfo {
      Status status = 1;
      map<string, SensorInfo> sensors = 2;
    }
```
?

Reply:
I just copied from `message Node` without any doubt. Thanks for the suggestion :1st_place_medal: